src/Draft.jl

```diff
@@ -1,113 +1,129 @@
 module Draft
-  import Pkg
-  import Suppressor
-  
-  export save_environment, @reuse, online, remove_persitance, silent, make_persitent
-
-  const SILENT = Ref(true)
-
-  # this function is executed upon loading the module
-  function __init__()
-    if Base.ACTIVE_PROJECT[] == nothing
-      Pkg.offline()
-      Pkg.activate(temp=true)
+
+import Pkg
+import Suppressor
+
+export @reuse
+export save_environment
+
+const SILENT = Ref(true)
+
+#
+# This function is executed upon loading the module, and will activate a new offline temporary environment. 
+#
+function __init__()
+    if isnothing(Base.ACTIVE_PROJECT[])
+        Pkg.offline()
+        Pkg.activate(temp = true)
     end
-  end
-
-  """
-  Save the current Project.toml into a target directory
-  """
-  function save_environment(target) 
-    filepath = joinpath(target,"Project.toml")
-    if !isfile(filepath)
-      if !isdir(target)
-        mkpath(target)
-      end
-      cp(Base.active_project(),filepath)
+end
+
+"""
+```
+save_environment(target::String; overwrite = false, switch = true)
+```
+
+Save the current `Project.toml` and `Manifest.toml` to a target directory, and switch (by default) the environment to the new created environment.
+
+`target` must be a directory path where the `Project.toml` file will be saved.  
+
+## Example
+
+```julia-repl
+julia> save_environment("/tmp/my_draft")
+  Activating project at `/tmp/my_draft`
+
+(my_draft) [offline] pkg>
+```
+
+"""
+function save_environment(target; overwrite = false, switch = true)
+    project = joinpath(target, "Project.toml")
+    manifest = joinpath(target, "Manifest.toml")
+    if !isfile(project) || overwrite
```

Comment:
Maybe this would be better
`!any(isfile.([project,manifest])) || overwrite`

Reply:
I think the current implementation solves all these issues, because the choice of overwritting or not is passed to the cp command. The function is also simpler as a whole. 

https://github.com/feanor12/Draft.jl/pull/2/commits/d4b2f648bea3b72da223237854d86d8fcf14ac32